compass/landice/tests/ensemble_generator/plot_ensemble.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python
+
+import numpy as np
+from matplotlib import pyplot as plt
+import netCDF4
+import glob
+import sys, os
+import yaml
+import xarray as xr
+import matplotlib.tri as tri
+
+targetYear = 20.0  # model year from start at which to calculate statistics
+labelRuns = True
+
+rhoi = 910.0
+rhosw = 1028.0
+
+runs = sorted(glob.glob("run*"))
+nRuns = len(runs)
+
+
+# obs from Rignot et al 2019 PNAS
+obs_discharge_thwaites = np.array([82.42,	83.25,	84.08,	84.90,	85.73,	86.56,	87.39,	88.22,	89.04,	89.87,	90.70,	91.40,	92.10,	92.80,	93.50,	94.20,	94.90,	95.60,	95.90,	96.20,	96.50,	96.80,	97.10,	97.40,	97.75,	98.10,	101.60,	101.40,	102.80,	104.30,	109.70,	114.10,	115.80,	118.70,	121.60,	123.80,	117.20,	115.10,	119.68])
+obs_discharge_haynes = np.array([10.10, 10.25, 10.40, 10.55, 10.70, 10.85, 11.00, 11.15, 11.30, 11.45, 11.60, 11.49, 11.37, 11.26, 11.14, 11.03, 10.91, 10.80, 10.93, 11.07, 11.20, 11.33, 11.47, 11.60, 11.35, 11.10, 12.20, 11.90, 12.50, 12.11, 12.80, 12.80, 12.90, 12.80, 13.90, 14.70, 12.30, 11.60, 12.53])
```

Comment:
Please reformat these to stick with the 79 character line length and with just one space between entries. Maybe consider having them in a data file and loading them instead of having them hard coded here.

Reply:
I've fixed for formatting, but left it hardcoded for now.  This script is run as a symlink from various places, so having this small amount of data hardcoded avoids issues of pointing to a data file.

Reply:
Hmm, if you can symlink the script, can't you symlink a data file in the same way? But I'm okay with you doing this your way.